<launch>
    <node name="c10pro_rtsp_ffmepg" pkg="network_port_camera" type="c10pro_socket_send.py" output="screen">
        <param name="host" value="192.168.144.108"/>
        <param name="port" value="5000"/>
        <param name="topic_sub_name" value="/c10pro/control_command"/>
    </node>
</launch>
